src/core/expression.js

```diff
@@ -355,7 +359,7 @@ function extractExponent(renderedTerms, index = 0) {
             index++;
         }
 
-        if (!term.elide) {
+        if (!term.elide && term.type !== types.operator) {
```

Comment:
**issue:** This is preventing the calculator from displaying any operator in a sub-expression placed as exponent. Moreover, what if the operator is a negative sign?

Try `9^(4+5)`
![image](https://github.com/oat-sa/tao-calculator-fe/assets/1500098/bd92079b-a438-4834-b381-8c795aaf78a6)

**suggestion:** You need to check if the operator is the first term of the exponent, and if this is not a negative sign...

Reply:
Thanks, fixed